<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>833</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>833</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabTalents">
    <attribute name="title">
     <string>Talents</string>
    </attribute>
    <widget class="QPushButton" name="buttonGenerateTalents">
     <property name="geometry">
      <rect>
       <x>692</x>
       <y>170</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tier 15</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tier 30</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tier 45</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tier 60</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tier 75</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tier 90</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tier 100</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxClass">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>90</y>
       <width>100</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSpec">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>140</y>
       <width>100</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>120</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Spec</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox1_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox2_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox3_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox4_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox5_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox6_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox7_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox6_2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>160</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox3_2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>70</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox2_2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>40</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox1_2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>10</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox4_2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>100</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox5_2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>130</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox7_2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>190</y>
       <width>175</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox5_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>130</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox2_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox4_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox6_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>160</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox1_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox7_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>190</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox3_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTalentsTabInfo">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>0</y>
       <width>100</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Class and spec</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxIgnoreZeros">
     <property name="geometry">
      <rect>
       <x>694</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ignore 0s</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabGear">
    <attribute name="title">
     <string>Gear</string>
    </attribute>
    <widget class="QPushButton" name="buttonGenerateGear">
     <property name="geometry">
      <rect>
       <x>692</x>
       <y>170</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddGear">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Gear</string>
     </property>
    </widget>
    <widget class="QListView" name="listViewGearList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRemoveGear">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Gear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSpec_3">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>140</y>
       <width>100</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>120</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Spec</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxClass_3">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>90</y>
       <width>100</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelGearInfo">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>0</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Class and spec for Gear options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonEditGear">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Gear</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabRelics">
    <attribute name="title">
     <string>Relics</string>
    </attribute>
    <widget class="QPushButton" name="buttonGenerateRelic">
     <property name="geometry">
      <rect>
       <x>692</x>
       <y>170</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddRelic">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Relic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRemoveRelic">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Relic</string>
     </property>
    </widget>
    <widget class="QListView" name="listViewRelicList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSpec_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>140</y>
       <width>100</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>120</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Spec</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxClass_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>90</y>
       <width>100</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelRelicInfo">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>0</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select Class and spec for Relic options&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="textEditAPL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>811</width>
     <height>331</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Please put base APL here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOutputFile">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>600</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Output File</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditFileSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Directory For File Here&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
